release/authorization/authorization.proto

```diff
@@ -0,0 +1,124 @@
+// Authorization for gnXi operations.
```

Comment:
@dplore @chrisy @aashaikh 

Would appreciate your thoughts here -- I think that this is OK to be in `openconfig/public` given that it is a data model that corresponds to multiple different gN*I protocols, but I am not sure that we have precedent here. Would you suggest that we include this here - or potentially put it in a different repo (if so, it'd be good to discuss where).

I think that it should likely at least be in `release/proto/authorization`.

Reply:
The placement seems a bit odd to me -- my preference would be to put it elsewhere.   Since we have a placeholder repo for gnsi already, that seems a more likely place for this.

Reply:
sure, wasn't the gnsi there though a different thing entirely?
(I have no care where this lands, to be clear... this was as good as any starting point)

Reply:
since that repo is empty, and this proto package is called `gnsi.authorization`, it seems like it could go there seamlessly :-)

Reply:
ah! ok, I can move this (I think) pretty directly to that part of the tree.
Wait on robjs/chrisy/dplore? or just move and pay later? :)

Reply:
Ddi the GNSI repository rename to GNPSI perhaps?
  openconfig/gnpsi/README.md:
  "Have single gNSI RPC's in the service and include typing in response"

Reply:
thanks -- I see the PR in the gnsi repo now.